packages/react-component-library/src/components/SectionDivider/SectionDivider.stories.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import { ComponentStory, ComponentMeta } from '@storybook/react'
+
+import { SectionDivider } from './index'
+
+export default {
+  component: SectionDivider,
+  title: 'Forms/Section Divider',
+  parameters: { actions: { argTypesRegex: '^on.*' } },
```

Comment:
Hmm Storybook has stopped displaying the `children` prop now.

Just tested and both of these work to get it back:

```tsx
 export default {
  // ...
  argTypes: {
    children: {
      type: 'string',
    },
  },
  // ...
}
```

or further down:

```tsx
Default.args = {
  children: '',
}
```